src/components/textarea/textarea.scss

```diff
@@ -1,5 +1,10 @@
 @import 'colors';
 
+.carbon-textarea .common-input__label {
```

Comment:
This is OK, but I'm wondering whether there's a better way to do this using BEM? e.g. perhaps using `.common-input__label--textarea` or `.carbon-textarea__label` instead of the descendant selector you are using.

Not sure how that'd work - or if it's possible - with the `InputLabel` decorator though. @andrewjtait might know?

Reply:
I think this is OK as it's using the parent to control the layout of the children, which is how `flex` and `grid` both work - I'm come to the conclusion that using inheritance is OK in CSS in this context